### DISCUSSION TITLE
Add shadow to announcement bar

### DISCUSSION THREAD

**Discussion by Samrose-Ahmed** [facebook/docusaurus#7827] [Q&A] [answered]
I'm having some difficulty adding a shadow to the Docusaurus announcement bar. It doesn't show, it seems its overlaid oddly with the subsequent navbar, but I can't quite tell what's going on. Has anyone added CSS to add shadow to the announcement bar?

**Comment by slorber** [comment, marked as answer]
Apparently, you have to add a z-index that makes it display on top of the navbar (which has a z-index)

This works: `z-index: calc(var(--ifm-z-index-fixed) + 1);`

<img width="1274" alt="CleanShot 2022-08-09 at 11 32 10@2x" src="https://user-images.githubusercontent.com/749374/183615685-e7fc9346-b9cb-4199-a946-67cdeba3444a.png">

I'll try to do this by default and replace our announcementBar border-bottom by a shadow

**Reply by Samrose-Ahmed** [reply]
Thanks eventually figured it out but forgot to mention.

**Reply by slorber** [reply]
https://github.com/facebook/docusaurus/pull/7940

**Reply by slorber** [reply]
FYI the z-index announcementBar change is reverted in https://github.com/facebook/docusaurus/pull/8163 because of https://github.com/facebookincubator/infima/issues/275
